poetry.lock

```diff
@@ -4422,20 +4465,6 @@ files = [
 [package.dependencies]
 types-urllib3 = "*"
 
-[[package]]
```

Comment:
Sames as the other dependabot PR #432. Any poetry pros know if this is a real problem? @betolink @jhkennedy  (I _think_ I remember you both saying you're familiar!)